spec.emu

```diff
@@ -649,8 +650,12 @@ contributors: Ujjwal Sharma, Younies Mahmoud
             <td>*"nanosecondsDisplay"*</td>
           </tr>
           <tr>
-            <td>[[FractionalDigits]]</td>
-            <td>*"fractionalDigits"*</td>
+            <td>[[MinimumFractionDigits]]</td>
+            <td>*"minimumFractionDigits"*</td>
+          </tr>
+          <tr>
+            <td>[[MaximumFractionDigits]]</td>
+            <td>*"maximumFractionDigits"*</td>
```

Comment:
Actually now I'm thinking to just keep `fractionalDigits` as a single standalone option. We haven't established a use case where minimumFractionDigits is useful except when it is either 0 or the same as maximumFractionDigits, and so if it is 0 by default, then we should be good to go.

Reply:
Yeah, the reason why I stuck to a single `fractionalDigits` option was that the use cases for specifying a max _and_ min are minimal and I think in most cases people just specify 0 or an integer number of fractional digits. That said, I'm not against this change and I think it helps with `NumberFormat` even if it's not absolutely necessary.